5.4.0 RN update with the backported tickets (#312)
hazelcast/management-center-docs: docs/modules/release-notes/pages/5-4-0.adoc

* Fixed an error on Persistence page when node limit is exceeded. [MC-2514]
* Fixed Map Journal enablement representation. [MC-2420]
* Fixed incorrect docs link in `hz-mc --help`. [MC-2401]

> * Fixed an incorrect total count in case of a stopped event emitter. [MC-2761]

Reviewer: ```suggestion
* Fixed an incorrect total count, which occurred following a stopped event emitter. [MC-2761]
```
